tests/0081-admin.c

```diff
@@ -1708,6 +1708,8 @@ static void do_test_DeleteConsumerGroupOffsets (const char *what,
                                   NULL, 0,
                                   15*1000);
 
+        rd_sleep(1); /* Additional wait time for cluster propagation */
```

Comment:
How long will it take for the cluster propagation usually? I'm wondering that will 1 additional second long enough?

Reply:
Fair question, and the answer is we can't really know.
On the line above we query the cluster for metadata on the topic, but that only represents a single broker's view, while other brokers may not yet have the full info.
The proper solution would be to query each broker in the cluster, but there's no easy way to do that with librdkafka.

So this arbitrary 1 second sleep is a naiive but cheap workaround that improves test stability.